src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorDocumentMappingService.cs

```diff
@@ -449,6 +450,31 @@ public async override Task<WorkspaceEdit> RemapWorkspaceEditAsync(WorkspaceEdit
             return workspaceEdit;
         }
 
+        public async override Task<Range> MapFromProjectedDocumentRangeAsync(Uri virtualDocumentUri, Range projectedRange, CancellationToken cancellationToken)
```

Comment:
Perhaps I just have Monday brain but the names of all the methods in this file don't really click with me. Like `MapFromProjectedDocumentRangeAsync(Uri virtualDocumentUri, Range projectedRange...`, it seems like we're using "virtual" and "projected" to mean the same thing in this case (ie .g.cs files or whatever) but then failing to describe what it is we're returning (I assume it's the range in the `.razor` document, but the method signature doesn't tell us).

Of course, that's a problem with every method in this file, and changing it would spider out to everywhere so don't worry about it here, I just wanted to get my thoughts on paper.

Reply:
Yeah, I agree, we should come up with a new name. I'd personally prefer "Generated" or something, as even virtual doesn't seem real enough. We can discuss and follow up

Reply:
Logged https://github.com/dotnet/razor-tooling/issues/6711